### ISSUE TITLE
error with a github action formatter error

### ISSUE THREAD

**Issue by staabm** [composer/composer#11750]
My `composer.json`:

any change in the composer.json without a updated composer.lock

When I run this command: 

```
composer validate
```

I get the following output: 

the error reported is not in github actions format, therefore you need to open the github actions log and look into the actual log

![grafik](https://github.com/ergebnis/composer-normalize/assets/120441/e825ade2-5196-4cfd-8441-09a13d8db396)

but I expected a github action formatted error, which would automatically show up in the files-changed tab.

example implementation for the github action error format can be found in `cs2pr`:
https://github.com/staabm/annotate-pull-request-from-checkstyle/blob/b824adfd29a6e6a900de976b5c893c62ce66edca/cs2pr#L131-L150

**Comment by alcohol** [comment]
I'm confused.. you want us to output errors by default in a Github action compatible format? Even though Github actions are not our primary target audience?

**Comment by staabm** [comment]
I suggested to feature detect github actions running and output errors in a format which integrate nicely with the CI (and github.com)

**Comment by alcohol** [comment]
Yeah that sounds nice, but it also opens up the door for "please support [gitlab|bitbucket|x]". Maybe that's trivial, maybe it is not. Submitting a PR is always an option though :-)

**Comment by staabm** [comment]
I don't know any other CI specific format :-). Will send a PR
